NAMESPACE

```diff
@@ -24,27 +24,44 @@ export(module_aglu_L205.ag_prodchange_cost_input_DISABLED)
 export(module_aglu_L2051.ag_prodchange_cost_input_irr_DISABLED)
 export(module_aglu_L2071.ag_water_irr_DISABLED)
 export(module_aglu_L221.land_input_1_DISABLED)
+<<<<<<< HEAD
```

Comment:
@CLynchy You can't (well, you can, but shouldn't!) commit files with conflicts in them. Happy to go over how to fix/resolve if useful.